packages/react/src/__test__/experimental/Await.spec.tsx

```diff
@@ -0,0 +1,111 @@
+import { act, render, screen, waitFor } from '@testing-library/react'
```

Comment:
It's not related to this PR, but I have a question!

Is it better to be in a folder with only a test code?
Isn't it better to be as close to the actual implementation as possible according to the principle of locality when maintaining? To look at the implementation and the test code together

Reply:
I understood you may mean removing __test__ directory is better
I'll remove __test__ and refactor it after this PR merging or close